app/resources/international_days.json

```diff
@@ -0,0 +1,463 @@
+﻿{
+    "calender": [
```

Comment:
This file seems to be unused, is it not needed?

Reply:
fixed